FreestarAds-Ogury/Source/objc/OguryBannerMediator.m

```diff
@@ -38,20 +37,27 @@ - (BOOL)canShowBannerAd {
     return YES;
 }
 
+- (BOOL)canShowInlineInviewAd {
+    return YES;
+}
+
 - (BOOL)isAdaptiveEnabled {
     return [Freestar adaptiveBannerEnabled];
 }
 
-- (NSString*)placementId {
-    return self.mPartner.placement_id;
+-(void)loadBannerAd {
+    [self loadInlineInviewAd];
 }
 
--(void)loadBannerAd {
+- (void)loadInlineInviewAd {
     FSTRLog(@"OGURY: loadBannerAd");
 
-    self.ad = [[OguryBannerAd alloc] initWithAdUnitId:[self placementId]];
-    [self.ad loadWithSize:self.requestedSize];
+    FSTRLog(@"OGURY: placement_id %@", self.mPartner.placement_id);
+    FSTRLog(@"OGURY: adunitId %@", [self.mPartner adunitId]);
+
+    self.ad = [[OguryBannerAd alloc] initWithAdUnitId:[self.mPartner adunitId]];
```

Comment:
Either add a nil check here to ensure you don't pass nil then call _partnerAdLoadFailed:_ or check for nil adunit id then call _partnerAdLoadFailed:_ and return early.

Reply:
Fixed